crates/polars-pipe/src/executors/sinks/ordered.rs

```diff
@@ -10,11 +12,15 @@ use crate::operators::{
 #[derive(Clone)]
 pub struct OrderedSink {
```

Comment:
If we want to add lifecycle parameter here and then change the `SchemaRef` to `&Schema`, it is also ok to me.

Reply:
On streaming queries a single schema clone is fine. They are meant for huge data sets. ^^